src/redux/Graphql/saga.js

```diff
@@ -154,6 +307,10 @@ export function* getTransactionsHandler() {
       yield put({
         type: actions.GET_TRANSACTIONS_RETURN,
         value: [],
+        error: {
```

Comment:
these background graphql calls dont need error objects since we fail silently, remember?